pypuf/simulation.py

```diff
@@ -113,29 +116,30 @@ class NoisyLTFArray(LTFArray):
     Class that simulates k LTFs with n bits and a constant term each 
     with noise effect and constant bias (=scale) added.
     """
-    def __init__(self, weight_array, transform, combiner, scale, noise):
+    def __init__(self, weight_array, transform, combiner, bias, scale, intensity=0.1):
         """
         Initializes LTF array with bias and noise effect based on inputs.
```

Comment:
`based on parameters` is better as inputs can be confused with the inputs to `ltf_eval`